docs/troubleshooting/large-files.mdx

```diff
@@ -11,31 +11,79 @@ When pushing your changes, you might see a message indicating that some files ex
 
 If you encounter an error like this, it's likely due to inadvertently committing files that are too large or not meant to be tracked (e.g., binaries or `node_modules/`).
 
+Depending on when these large files entered your Git history, there are two ways to resolve this.
 
-## Instructions
-Follow these steps to remove the large files:
+## Solution 1
```

Comment:
@glokta1 thoughts on naming this section "Step 1: Amend last commit" and "Step 2: Amend historical commits"? Worried that solution 1 / solution 2 might make it sound like it's an either/or situation